### PR TITLE
Add reporting and operator test fixes for gatekeeper

### PR DESCRIPTION
**Pull request by marzianor9**

### CODE REVIEW

File: cnf-tests/testsuites/e2esuite/gatekeeper/gatekeeper.go [hunk 1]
```diff
@@ -694,6 +714,21 @@ var _ = Describe("gatekeeper", func() {
...
+					Name: "gatekeeper-mutating-webhook-configuration",
+				},
+			}
+			mwConfigKey, err := k8sClient.ObjectKeyFromObject(mutatinWebhookConfiguration)
+			Expect(err).ToNot(HaveOccurred())
+
+			Eventually(func() bool {
+				err := client.Get(context.Background(), mwConfigKey, mutatinWebhookConfiguration)
+				Expect(err).ToNot(HaveOccurred())
+				return mutatinWebhookConfiguration.Webhooks[0].NamespaceSelector != nil &&
```
Comment (SchSeba) [inline cnf-tests/testsuites/e2esuite/gatekeeper/gatekeeper.go:728]: add a check for len of webhook is equal to 1 or you can have a panic here

Comment (marzianor9) [reply cnf-tests/testsuites/e2esuite/gatekeeper/gatekeeper.go]: Added

**Comment by SchSeba** [comment]
lgtm

@fedepaol any comments here?

**Pull request merged**